bower.json

```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "medium-editor": "^5.0.0",
     "to-markdown": "^2.0.1",
-    "uglify-js": "^2.5.0"
+    "UglifyJS2": "https://github.com/mishoo/UglifyJS2.git#^2.5.0"
```

Comment:
Without the url, simply "  "uglify-js": "^2.5.0"" wasn't working for me, didn't seem to find it on bower. Ok for devDependencies, I did the same for node, uglify and barbe went to devdependencies